### ISSUE TITLE
Something may be wrong with the document--plugins/jwt

### ISSUE THREAD

**Issue by lexburner** [Kong/kong#3393]
I am using the jwt plugin in **Kong**，this section took me a lot of time
(https://getkong.org/plugins/jwt/#optional-verified-claims)
I want to Verified claims(exp) during jwt authentication, follow the doc

```
curl -X PATCH http://kong:8001/routes/{route id}/plugins/{jwt plugin id} \
    --data "config.claims_to_verify=exp,nbf"
```
but i got the message:

{"message":"Not found"}

i try like this 

```
curl -X PATCH http://kong:8001/plugins/{jwt plugin id} \
    --data "config.claims_to_verify=exp,nbf"
```
it works.

i guess that the apis may have some changes in v0.13.0 but the document doesn't sync?

**Comment by thibaultcha** [comment]
Hi,

Thanks for the report. We would welcome a PR for this, or else we'll fix it as soon as we can manage!

**Comment by lexburner** [comment]
well,let me try

**Comment by bungle** [comment]
Yes it is a bug in documentation, the new API does nest that deep. I am not sure if we want it one level deeper but we need to stop at some point.

**Comment by bungle** [comment]
What I mean is that:

```
curl -X PATCH http://kong:8001/routes/{route id}/plugins/{jwt plugin id}
```

In that the `/routes/.../` doesn't add any relevant information, at least for `PATCH`.
